src/bin/main.rs

```diff
@@ -34,6 +34,10 @@ pub struct ProgramOptions {
     /// Testing mode?
     #[arg(long)]
     pub test_mode: bool,
+
+    /// Overwrite existing Lampe project if it exists.
```

Comment:
```suggestion
    /// Overwrite all files if they exist, even if they were not generated by Lampe. Note that generated files are able to be overwritten by default.
```

Please wrap my suggestion properly!